File: 백준/Bronze/1100. 하얀 칸/하얀 칸.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int cnt = 0;

        for (int i = 0; i < 8; i++) {
            char[] li = br.readLine().toCharArray();
            for (int j = 0; j < 8; j++) {
                if (i%2 == 0 && j%2 == 0 && li[j] == 'F') cnt ++;
                else if (i%2 == 1 && j%2 == 1 && li[j] == 'F') cnt ++;
            }
        }
        System.out.println(cnt);
    }
}

File: 백준/Bronze/1100. 하얀 칸/하얀 칸.py
import sys
input = sys.stdin.readline
cnt = 0

for i in range(8):
    li = list(input())
    for j in range(8):
        if not i%2 and not j%2 and li[j] == "F": cnt += 1
        elif i%2 and j%2 and li[j] == "F": cnt += 1
print(cnt)